4-fleetscope/modules/env_baseline/README.md

```diff
@@ -13,7 +13,7 @@ The following resources are created:
 |------|-------------|------|---------|:--------:|
 | cluster\_membership\_ids | The membership IDs in the scope | `list(string)` | n/a | yes |
 | env | The environment to prepare (ex. development) | `string` | n/a | yes |
-| fleet\_project\_id | The fleet project ID | `string` | n/a | yes |
+| fleet\_project\_id | The Fleet project ID | `string` | n/a | yes |
```

Comment:
Given the prevalence of `fleet`, we should probably be consistent with caps or no-caps.  From the docs usage I would vote for lowercase: https://cloud.google.com/kubernetes-engine/docs/fleets-overview

Reply:
yes, agreed, switched to lowercase.